src/webots/core/WbApplicationInfo.cpp

```diff
@@ -33,6 +37,24 @@ const WbVersion &WbApplicationInfo::version() {
   return webotsVersion;
 }
 
+const QString &WbApplicationInfo::branch() {
+  static QString version;
```

Comment:
`version` sounds bit misleading to me, why not calling it `branchName` instead?
```suggestion
  static QString branchName;
```